Continuous test failing because of missing dependency `six`

Most continuous tests are failing with the following error message:
```
Traceback:
/usr/local/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/unit/test_retry.py:17: in <module>
    from google.cloud.storage import _helpers
google/cloud/storage/__init__.py:35: in <module>
    from google.cloud.storage.batch import Batch
google/cloud/storage/batch.py:27: in <module>
    import six
E   ModuleNotFoundError: No module named 'six'
```

--- a/setup.py
+++ b/setup.py
@@ -38,6 +38,7 @@ dependencies = [
     "google-resumable-media >= 1.3.0, < 3.0dev; python_version>='3.6'",
     "requests >= 2.18.0, < 3.0.0dev",
     "googleapis-common-protos < 1.53.0; python_version<'3.0'",
+    "six",
 ]
 extras = {}
 

--- a/setup.py
+++ b/setup.py
@@ -37,6 +37,7 @@ dependencies = [
     "google-resumable-media >= 1.3.0, < 2.0dev; python_version<'3.0'",
     "google-resumable-media >= 1.3.0, < 3.0dev; python_version>='3.6'",
     "requests >= 2.18.0, < 3.0.0dev",
+    "protobuf < 3.18.0; python_version<'3.0'",
     "googleapis-common-protos < 1.53.0; python_version<'3.0'",
     "six",
 ]

